### ISSUE TITLE
Usage with older Fable compiler

### ISSUE THREAD

**Issue by spingee** [dbrattli/Fable.Reaction#95]
Hi,
I tried to add Autocomplete example to my app which was based on safe stack template, it didnt work because it cant find some compiled js , see the warning from console:

`WARNING in ./src/Client/Views/Autocomplete.fs 177:23-81
"export 'Reaction$$$useStatefulStream$$5E046E25' (imported as 'Reaction$0024$0024$0024useStatefulStream$0024$00245E046E25') was not found in '../../../.fable/Fable.Reaction.3.5.0/Reaction.fs'
 @ ./src/Client/Views/Index.fs
 @ ./src/Client/App.fs
 @ ./src/Client/Client.fsproj`

when i run autocomplete example from this repo it will generate *.js in .fable/Fable.Reaction.3.5.0 but not in my app, there are no js files.
So is this library only usable with fable compiler 3.0 and my only option is upgrade my app to it? Or is there some "workaround".
Thanks

**Comment by spingee** [comment]
Or perhaps its .net5 related problem?

**Comment by dbrattli** [comment]
I think it's an issue with the `[Erase]` attribute used on https://github.com/dbrattli/Fable.Reaction/blob/main/src/Reaction.fs#L23

We should probably just remove it since it's causing these problems.

**Comment by spingee** [comment]
Yes, i removed it from file in .fable directory and it started to work

**Comment by dbrattli** [comment]
Nice, please feel free to submit a PR
